fc-examine/schema.go

```diff
@@ -0,0 +1 @@
+../tvx/schema.go
```

Comment:
is this some Go magic trick I don't know about? 😮

Reply:
a git symbolic link :)